src/main/java/com/esl/demo/service/implementations/ESLServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package com.esl.demo.service.implementations;
+
+import com.esl.demo.dto.ESLDto;
+import com.esl.demo.service.interfaces.ESLService;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
```

Comment:
Why add transactional annotation here?

Reply:
So that if you write in the DB two times in the same method, if the second write fails, the first will be canceled.

Reply:
Can you give me an example here how this risk/issue manifests?